app/member-reminders/member-reminders-public/src/main/kotlin/com/hedvig/android/memberreminders/GetMemberRemindersUseCase.kt

```diff
@@ -123,4 +132,8 @@ sealed interface MemberReminder {
     val contractId: String,
     override val id: String = UUID.randomUUID().toString(),
   ) : MemberReminder
+
+  data class ContactInfoUpdateNeeded(
+    override val id: String = UUID.randomUUID().toString(),
+  ) : MemberReminder
```

Comment:
```suggestion
  data object ContactInfoUpdateNeeded : MemberReminder {
    override val id: String = UUID.randomUUID().toString()
  }
```
I wonder, would a syntax like this work here? Since we never pass in a custom ID, and we just need it to be unique but we won't ever really have 2 of these reminders in a list of reminders.